front/app/containers/Admin/pagesAndMenu/containers/ContentBuilder/components/CraftComponents/Highlight/index.tsx

```diff
@@ -0,0 +1,298 @@
+import React from 'react';
+
+import {
+  Box,
+  colors,
+  useBreakpoint,
+  Text,
+  defaultCardStyle,
+  media,
+  Title,
+  Input,
+} from '@citizenlab/cl2-component-library';
+import { useEditor, useNode } from '@craftjs/core';
+import { Multiloc } from 'component-library/utils/typings';
+import styled, { useTheme } from 'styled-components';
+
+import useLocalize from 'hooks/useLocalize';
+
+import { DEFAULT_PADDING } from 'components/admin/ContentBuilder/constants';
+import Button from 'components/UI/Button';
+import InputMultilocWithLocaleSwitcher from 'components/UI/InputMultilocWithLocaleSwitcher';
+
+import { useIntl } from 'utils/cl-intl';
+import { removeUrlLocale } from 'utils/removeUrlLocale';
+
+import messages from './messages';
+
+const Container = styled.div`
```

Comment:
Styled components are needed here to more cleanly specify the breakpoints styles.